### ISSUE TITLE
[HelpHub] Feedback on Comparing Patterns, Template Parts, and Reusable Blocks

### ISSUE THREAD

**Issue by zzap** [WordPress/Documentation-Issue-Tracker#906]
## Issue Description

Originally reported by https://profiles.wordpress.org/richardpatterson/

> The sentence: “These carry semantic meaning within block themes so it’s important to make the distinction.” is not at all clear. Surely there is another less obscure way to say what you mean.

## URL of the Page with the Issue

https://wordpress.org/documentation/article/comparing-patterns-template-parts-and-reusable-blocks/

## Section of Page with the issue

https://wordpress.org/documentation/article/comparing-patterns-template-parts-and-reusable-blocks/#repeated-structure-across-your-site

## Why is this a problem?

It's not following documentation style guide in [style, voice and tone](https://make.wordpress.org/docs/style-guide/general-guidelines/style-voice-tone/).

## Suggested Fix

Rephrase the sentence.

**Comment by chekle** [comment]
I agree that the sentence could be worded a little more clearly. Here's what I came up with and posted and updated the page with:

For anything that’s more about site structure that’s repeated across your site, like a Header or Footer, it’s best to use a template part. These are intended for structural elements in a block theme, so it’s important to distinguish them from patterns.

Closing this one.
